libs/application/templates/parental-leave/src/fields/EditPeriodsReview/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC } from 'react'
+
+import { Application } from '@island.is/application/core'
+import { Box } from '@island.is/island-ui/core'
+
+import { useLocale } from '@island.is/localization'
+
+import Timeline from '../components/Timeline'
+import { formatPeriods, getExpectedDateOfBirth } from '../../parentalLeaveUtils'
+import { Period } from '../../types'
+
+import { parentalLeaveFormMessages } from '../../lib/messages'
+
+interface ReviewScreenProps {
+  application: Application
+  goToScreen?: (id: string) => void
+  editable?: boolean
+}
+
+const Review: FC<ReviewScreenProps> = ({ application }) => {
+  const { formatMessage } = useLocale()
+
+  const dob = getExpectedDateOfBirth(application)
+
+  if (!dob) {
+    return null
```

Comment:
Yeah we have this check through out the screens, ideally we should never hit it at this point in the flow. I've refactored it to show a message in case the `dob` is not available.